Sources/SwiftKuery/Column.swift

```diff
@@ -16,8 +16,26 @@
 
 // MARK: Column
 
-/// Definition of table column.
+/**
+ The `Column` class is used to represent a single column in an SQL table in Swift.
+ A combination of columns are used to construct a `Table` class which matches a specific table in an SQL database.
+ The `Column` class details the column name, the table the column belongs to, any SQL keywords which apply to the column and the data type of the column.
+ ### Usage Example: ###
+ In this example, a person `Table` class matching a table stored in an SQL database is defined.
```

Comment:
I think we can cut this right down to - "In this example, a person `Table` class, containing three instances of the `Column` class is defined."